src/index.ts

```diff
@@ -25,6 +25,22 @@ export interface Collection<K, V> extends Map<K, V> {
 export class Collection<K, V> extends Map<K, V> {
 	public static readonly default: typeof Collection = Collection;
 
+	/**
+	 * Obtains an element if the key exists, otherwise sets and returns the value provided by the default value generator.
+	 *
+	 * @param key The key to get from if it exists or set otherwise
+	 * @param defaultValueGenerator A function that returns the ensured value
```

Comment:
```suggestion
	 * @param key The key to get if it exists, or set otherwise
	 * @param defaultValueGenerator A function that generates the value to be set if it's missing
```